From 8ab120f6ad6629e826ded6d8f910c91f441bafd4 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 6 Nov 2013 12:36:02 +0100
Subject: [PATCH 67/81] monitor: Remove pci_del command for Red Hat Enterprise Linux

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <6e7cc8214fdc2fd6a95a7446c667bf36fc13f3ce.1383741033.git.mrezanin@redhat.com>
Patchwork-id: 55517
O-Subject: [RHEL7 qemu-kvm PATCH v2 2/4] monitor: Remove pci_del command for Red Hat Enterprise Linux
Bugzilla: 1010858
RH-Acked-by: Michal Novotny <minovotn@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

This is forwardport of RHEL-6 commit 609b9f16a7443e86e55a3cdc76a484253e50267b:

    monitor: Remove pci_del command for Red Hat Enterprise Linux

    pci_del is the old way of hot-unplugging pci (nic or storage) devices.
    The newer device_del is the recommended and suggested command.

    Signed-off-by: Amit Shah <amit.shah@redhat.com>

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index d5ec349..14ba536 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1105,7 +1105,7 @@ STEXI
 Hot-add PCI device.
 ETEXI
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "pci_del",
         .args_type  = "pci_addr:s",
-- 
1.7.1

